pages/login/index.js

```diff
@@ -0,0 +1,120 @@
+import Cookies from "cookies";
+import Iron from "@hapi/iron";
+import { ENC_KEY } from "../api/login";
+import { useRouter } from "next/router";
+import Button from "../../components/ui/Button";
+import LoginForm from "../../components/login/LoginForm";
+import RegistrationForm from "../../components/login/RegistrationForm";
+import { useState } from "react";
+
+function login(props) {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [user, setUser] = useState("");
+    const [pwd, setPwd] = useState("");
+    const [pageState, setPageState] = useState("login");
+
+    const router = useRouter();
+
+    async function handleLogout(event) {
+        event.preventDefault();
+
+        await fetch("/api/logout", {
+            method: "POST",
+            body: JSON.stringify({
+                logout: true
+            }),
+            headers: {
+                "Content-Type": "application/json"
+            }
+        });
+        router.push("/login");
+    }
+
+    async function handleSubmit() {
+        await fetch("/api/login", {
+            method: "POST",
+            body: JSON.stringify({
+                username,
+                password
+            }),
+            headers: {
+                "Content-Type": "application/json"
+            }
+        });
+        router.push("/login");
+    }
+
+    async function handleRegistration(event) {
+        event.preventDefault();
```

Comment:
Got an error message that complained about event.preventDefault. tried removing it and it work fine without.

Reply:
Resolved